applications/ParticleMechanicsApplication/custom_strategies/strategies/MPM_residual_based_newton_raphson_strategy.hpp

```diff
@@ -456,49 +449,53 @@ class MPMResidualBasedNewtonRaphsonStrategy
         TSystemVectorType& mDx = *mpDx;
         TSystemVectorType& mb = *mpb;
         DofsArrayType& rDofSet = pBuilderAndSolver->GetDofSet();
-        //initializing the parameters of the Newton-Raphson cicle
+        
+        // Initializing the parameters of the Newton-Raphson cicle
         unsigned int iteration_number = 1;
         BaseType::GetModelPart().GetProcessInfo()[NL_ITERATION_NUMBER] = iteration_number;
-        //std::cout<<"initializing the parameters of the Newton-Raphson cicle"<<std::endl;
-        //BaseType::GetModelPart().GetProcessInfo().SetNonLinearIterationNumber(iteration_number);
         bool is_converged = false;
-        //bool ResidualIsUpdated = false;
+        
         pScheme->InitializeNonLinIteration(BaseType::GetModelPart(), mA, mDx, mb);
-        //std::cout<<"InitializeNonLinIteration"<<std::endl;
+        
         is_converged = mpConvergenceCriteria->PreCriteria(BaseType::GetModelPart(), rDofSet, mA, mDx, mb);
-        //std::cout<<"PreCriteria"<<std::endl;
-        //std::cout<<"is_converged"<<is_converged<<std::endl;
-        //function to perform the building and the solving phase.
-        //std::cout<<"mRebuildLevel"<<BaseType::mRebuildLevel<<std::endl;
-        //std::cout<<"mStiffnessMatrixIsBuilt"<<BaseType::mStiffnessMatrixIsBuilt<<std::endl;
+        
+        if (this->GetEchoLevel() >= 3)
+        {
+            KRATOS_INFO("MPM_Strategy") << "PreCriteria" << std::endl;
```

Comment:
KRATOS_INFO_IF, and maybe is better to put it everything in only one LOg

Reply:
Will modify!